Messenger/AppDelegate.mm

```diff
@@ -163,7 +164,9 @@ - (void)updateWindowTitlebar {
   auto updateButton = ^(NSView* buttonView) {
     auto buttonFrame = buttonView.frame;
     buttonFrame.origin.y = round((kTitlebarHeight - buttonFrame.size.height) / 2.0);
+    if (fullScreen) buttonFrame.origin.y -= 15;
```

Comment:
Can you please change this to not use a constant but instead `kTitlebarHeight `?